### PR TITLE
feat: update demo

### LINKED ISSUE #1020: Update demo doc
* remove `PERFORMANCE_SENSITIVE`
* update cluster mode usage

### PR DESCRIPTION
**Pull request by dl239**

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -24,25 +24,99 @@ w2 as (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN
...
+max(pickup_latitude) OVER w2 AS pc_max_pl,
+min(pickup_latitude) OVER w2 AS pc_min_pl,
+avg(pickup_latitude) OVER w2 AS pc_avg_pl,
+count(vendor_id) OVER w2 AS pc_cnt,
+count(vendor_id) OVER w AS vendor_cnt
+FROM t1
+WINDOW w AS (PARTITION BY vendor_id ORDER BY pickup_datetime ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW),
+w2 AS (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW);
+```
+:bulb: Note that:
```
Comment (jingchen2222) [inline demo/README.md:102]: Should we add a notice that users had better deploy SQL before `LOAD DATA` under online execute mode?

File: demo/predict-taxi-trip-duration-nb/script/train.py [hunk 2]
```diff
@@ -15,27 +15,19 @@
...
-    sql = fd.read()
-
 # run batch sql and get instances
-spark = SparkSession.builder.appName("OpenMLDB Demo").getOrCreate()
-parquet_train = "file:////work/taxi-trip/data/taxi_tour_table_train_simple.snappy.parquet"
-train = spark.read.parquet(parquet_train)
-train.createOrReplaceTempView("t1")
-train_df = spark.sql(sql)
-df = train_df.toPandas()
+df = pd.read_csv(args.feature_path);
```
Comment (tobegit3hub) [inline demo/predict-taxi-trip-duration-nb/script/train.py]: Need to aggregate the csv file in the path after offline import.

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: demo/README.md [hunk 3]
```diff
@@ -24,25 +24,100 @@ w2 as (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN
 ```
 
 ## 2. Demo with The Cluster Mode
-
 > :warning: Required docker engine version >= 18.03
-
-```bash
-# Pull the docker and start it
```
Comment (zhanghaohit) [inline demo/README.md:31]: add a newline after it

File: demo/README.md [hunk 4]
```diff
@@ -24,25 +24,100 @@ w2 as (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN
...
+```bash
+# Start the OpenMLDB CLI for the cluster mode
+../openmldb/bin/openmldb --zk_cluster=127.0.0.1:2181 --zk_root_path=/openmldb --role=sql_client
+```
+```sql
+# The below commands are executed in the CLI
+> CREATE DATABASE demo_db;
+> USE demo_db;
+> CREATE TABLE t1(id string, vendor_id int, pickup_datetime timestamp, dropoff_datetime timestamp, passenger_count int, pickup_longitude double, pickup_latitude double, dropoff_longitude double, dropoff_latitude double, store_and_fwd_flag string, trip_duration int, INDEX(ts=pickup_datetime));
+```
```
Comment (zhanghaohit) [inline demo/README.md:47]: should the index be put like this?
```
index(key=vendor_id, ts=pickup_datetime),                                                                                                                                                                                          
index(key=passenger_count, ts=pickup_datetime) 
```

**Review by zhanghaohit — DISMISSED** [review]

File: demo/README.md (hunk 4 above)
Comment (dl239) [reply demo/README.md:47]: OpenMLDB will create index automatically when executing deploy command

Comment (zhanghaohit) [reply demo/README.md:47]: what if:
- users specify the detailed index info during `CREATE TABLE`, e.g., `index(key=vendor_id, ts=pickup_datetime)`. Is it also ok?
- users doesn't specify any index info, i.e., no `INDEX()` part. Does it still auto create during `deploy`?

Comment (dl239) [reply demo/README.md:47]: yes

Comment (tobegit3hub) [reply demo/README.md:47]: Then should we simplify the SQL in demo. Just `CREATE TABLE t1(id string, vendor_id int, pickup_datetime timestamp, dropoff_datetime timestamp, passenger_count int, pickup_longitude double, pickup_latitude double, dropoff_longitude double, dropoff_latitude double, store_and_fwd_flag string, trip_duration int);`.

File: demo/predict-taxi-trip-duration-nb/script/predict_server.py [hunk 5]
```diff
@@ -42,6 +39,8 @@
 	("trip_duration", "int"),
 ]
 
+url = "http://127.0.0.1:8080/dbs/demo_db/deployments/demo"
```
Comment (tobegit3hub) [inline demo/predict-taxi-trip-duration-nb/script/predict_server.py]: The default port of apiserver in cluster mode should be change to 9080.

File: demo/README.md (hunk 4 above)
Comment (tobegit3hub) [reply demo/README.md:47]: Need to add `OPTIONS(partitionnum=1)` to deploy now.

**Review by lumianph — APPROVED** [review]

File: demo/README.md [hunk 6]
```diff
@@ -24,25 +24,101 @@ w2 as (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN
...
+> LOAD DATA INFILE 'file:///work/taxi-trip/data/taxi_tour_table_train_simple.csv' INTO TABLE t1 options(format='csv', header=true, mode='append');
+# You can see job status by the below command
+> show jobs;
+```
+**Start HTTP service for inference with OpenMLDB**
+```bash
+./start_predict_server.sh 127.0.0.1:9080 /tmp/model.txt
+```
+**Run inference with HTTP request**
+```bash
```
Comment (jingchen2222) [inline demo/README.md:121]: b: To read more details about the cluster mode, please refer to the [QuickStart (Cluster Mode)](https://github.com/4paradigm/OpenMLDB/blob/main/docs/en/cluster.md)
change link to docs.openmldb.ai

File: demo/predict-taxi-trip-duration-nb/Dockerfile [hunk 7]
```diff
@@ -3,13 +3,14 @@ FROM openjdk:11.0.13-jre-slim-bullseye
 LABEL org.opencontainers.image.source https://github.com/4paradigm/OpenMLDB
 
 RUN  apt-get update \
-     && apt-get install -y --no-install-recommends libgomp1 curl binutils procps python3 python3-pip python3-numpy \
+     && apt-get install -y --no-install-recommends libgomp1 curl binutils procps python3 python3-pip python3-numpy vim \
```
Comment (aceforeverd) [inline demo/predict-taxi-trip-duration-nb/Dockerfile:6]: no vim !

Comment (dl239) [reply demo/predict-taxi-trip-duration-nb/Dockerfile:6]: @tobegit3hub advice adding vim for user editing conf

Comment (aceforeverd) [reply demo/predict-taxi-trip-duration-nb/Dockerfile:6]: if a vimer, just install manually.  there is nano or vi already

Comment (aceforeverd) [reply demo/predict-taxi-trip-duration-nb/Dockerfile:6]: well, if vim is used in our quick start, then ..

**Pull request merged**